### DISCUSSION TITLE
将github issues导入project-graph

### DISCUSSION THREAD

**Discussion by Littlefean** [graphif/project-graph#133] [功能请求]
每个issues之间可能有着顺序依赖关系，从而形成一个拓扑图

但只有public的仓库才能爬，private的仓库好像爬不了

这样就像一个高级的、带有拓扑顺序的 todolist

**Comment by zty012** [comment]
纠正：用github apps可以让用户登录自己的账号，private仓库也能获取

**Comment by zty012** [comment]
- [x] 登录github账号
- [ ] 导入issues
